tools/publishIsolate.ts

```diff
@@ -0,0 +1,62 @@
+import fs from 'fs';
+import path from 'path';
+import childProcess from 'child_process';
+import getNpmPackageVersion from 'get-npm-package-version';
+import { compareVersions } from 'compare-versions';
+
+async function main() {
+  const isolatePackageRoot = './dist/isolate';
+
+  const publishPackageList: { packageName: string; dir: string, version: string }[] = [];
+
+  // get directory listing
+  const files = fs.readdirSync(isolatePackageRoot);
```

Comment:
I think new version should managed by `isolate.config.json` and also read from it.